cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -1667,131 +1667,6 @@ func TestUpdateSecrets(t *testing.T) {
 				},
 			},
 		},
-		{
```

Comment:
Why did we only delete test cases?

Reply:
Those test cases expecting an error for the immutable field. I will add a test case that makes sure that the re-creating is happening correctly

Reply:
updated.